test/core.js

```diff
@@ -370,15 +370,63 @@ QUnit.test( "jQuery.holdReady (warn only)", function( assert ) {
 	} );
 } );
 
+QUnit[ jQueryVersionSince( "3.1.1" ) ? "test" : "skip" ]( "jQuery.trim", function( assert ) {
+	assert.expect( 26 );
+
+	var nbsp = String.fromCharCode( 160 );
+
+	expectWarning( assert, "jQuery.trim", 1, function() {
```

Comment:
I split it like that as Migrate now only warns once against any API used twice... Maybe we should have a test helper here so that we can track how many times a warning is detected even if it's skipped?

Reply:
I'm fine with separate tests really. In real code once you know you're looking for `.trim` you can just search for that.

Reply:
@dmethvin I submitted #332 acting on the idea proposed in my comment above.